src/cmd/linuxkit/run_qemu.go

```diff
@@ -68,6 +73,7 @@ func runQemu(args []string) {
 	fw := flags.String("fw", "/usr/share/ovmf/bios.bin", "Path to OVMF firmware for UEFI boot")
 
 	// VM configuration
+	enableKVM := flags.Bool("enable-kvm", haveKVM(), "Enable or disable QEMU")
```

Comment:
The "QEMU" option is `-enable-kvm` although I confess that wasn't a consideration when I typed it above and it doesn't have the same `=false` ability, so compat is probably not the biggest concern.

I'll change.